sky/utils/kubernetes_enums.py

```diff
@@ -0,0 +1,29 @@
+"""Kubernetes enums for SkyPilot."""
```

Comment:
Yes, we need to have the enums outside of `kubernetes_utils` because this will lead to many circular imports. e.g.
`sky.skypilot_config -> sky.utils.common_utils -> sky.utils.schemas -> sky.utils.kubernetes_untils -> sky.utils.backend_utils -> sky.skypilot_config`